docs/src/components/sidebarIndex.js

```diff
@@ -31,6 +31,11 @@ const sidebarIndex: sidebarIndexType = [
     sectionPathname: 'setup',
     pages: ['GestaltProvider'],
   },
+  {
+    sectionName: 'Accessibility',
```

Comment:
Should this be "Hooks" or similar? We don't have too many a11y components.

Reply:
@chrislloyd Just wait for it, this is only the beginning... Also plan on exporting `useFocusVisible` and adding `VisuallyHidden` + `useId` soon